Task-WebRTC.md

```diff
@@ -136,11 +138,15 @@ number.
 
 # Signalling Channel Handover
 
-As soon as both clients have exchanged the required messages and the
-WebRTC `RTCPeerConnection` instance informs the user application that
-the peer-to-peer connection setup is complete, the user application
-SHOULD request that the client hands over the signalling channel to a
-dedicated data channel:
+As soon as the user application has created a WebRTC `RTCPeerConnection`
+instance which intends to use this task for its signalling, the user
+application SHOULD request that the client hands over the signalling
+channel to a dedicated data channel before *offer* or *answer* of the
```

Comment:
Why this constraint? A data channel can only be established once the peer connection is up. Depending on the WebRTC implementation, an offer might be created as soon as the `RTCPeerConnection` is established.

Reply:
https://github.com/saltyrtc/saltyrtc-task-webrtc-js/commit/d14c294b6b617aad4ca3e647e3b16d02a10ef4b6 That's all it took.